src/styles/paragraphs/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Paragraphs
+description: Use paragraphs for your core content
+section: Styles
+theme: Typography
+layout: layout-pane.njk
+showPageNav: true
+order: 8
+---
+
+{% from "_example.njk" import example %}
+
+The default paragraph font size is 19px on large screens and 16px on small screens.
```

Comment:
```suggestion
The default paragraph font size is 19px.
```

Updated to be in line with the new scale – should we say anything else here to say that this has recently changed and link to an explanation? This might be a pattern we repeat on other pages where the values have changed...

Reply:
Agree. Is there anything we can take from the WCAG 2.2 work here?

Reply:
Maybes. Summat to chat about next week!

Reply:
I'm not sure about WCAG 2.2. Would this work? [text spacing](https://www.w3.org/TR/WCAG22/#text-spacing)

Reply:
chat with Charlotte about using the WCAG thing
mock it up: one with wcag banner, one using just text